test/packages/parallel/gcp/data_stream/compute/_dev/test/system/test-default-config.yml

```
@@ -0,0 +1,6 @@
+# wait_for_data_timeout: 10m
+vars:
+  project_id: "{{GCP_PROJECT_ID}}"
+  credentials_json: "{{GOOGLE_CREDENTIALS}}"
```

```suggestion
  credentials_json: '{{{GOOGLE_CREDENTIALS}}}'
```

Because of HTML escaping from https://github.com/aymerick/raymond#html-escaping.